config/helm/chart/default/templates/Common/customresource-dynakube.yaml

```diff
@@ -26,10 +26,14 @@ metadata:
     operator.dynatrace.com/feature-disable-oneagent-readonly-host-fs: "true"
     {{- end }}
     {{- if (.Values.activeGate).apparmor }}
-    alpha.operator.dynatrace.com/feature-activegate-apparmor: "true"
+    operator.dynatrace.com/feature-activegate-apparmor: "true"
     {{- end }}
     {{- if (.Values.activeGate).readOnlyFs }}
-    alpha.operator.dynatrace.com/feature-activegate-readonly-fs: "true"
+    operator.dynatrace.com/feature-activegate-readonly-fs: "true"
+    {{- end }}
+    {{- if and (.Values.activeGate.autoApiMonitoring) (has "kubernetes-monitoring" .Values.activeGate.capabilities) }}
+    operator.dynatrace.com/feature-automatic-kubernetes-api-monitoring: "true"
+    operator.dynatrace.com/feature-disable-metadata-enrichment: "false"
```

Comment:
@mjgrzybek it was included in the dynakube.yml template that was downloadable when following the guide to deploy from the UI. However I now noticed that it has since been removed, so I shall amend this PR.